doc/operator-creation.md

```diff
@@ -128,7 +128,20 @@ There are a few things to know and (try to) understand while developing operator
    the shared underlying subscription.
    - When you `subscribe` to an Observable, the functions or Observer you passed are used to create the final `destination`
    `Subscriber` for the chain. It's this `Subscriber` that is really also the shared `Subscriptoin` for the operator chain.
-   
+
+### Submitting a PR for an Operator
+
+Please complete these steps for each new operator added to RxJS as a pull request:
```

Comment:
Perhaps we could change the language here to be a little more explicit. Rather that "it should, it should", etc. We could use "It MUST including" and "Consider including". "Should" is a little ambiguous between "this is optional" and "this is not optional".